src/backend/distributed/planner/query_pushdown_planning.c

```diff
@@ -53,6 +53,15 @@ typedef enum RecurringTuplesType
 	RECURRING_TUPLES_RESULT_FUNCTION
 } RecurringTuplesType;
 
+/*
+ * FlattenJoinVarsMutatorContext stores context variable needed by
+ * FlattenJoinVarsMutator function.
+ */
+typedef struct FlattenJoinVarsMutatorContext
+{
+	Query *queryTree;
+} FlattenJoinVarsMutatorContext;
```

Comment:
Do we need the struct at all? I thought we'd only pass `queryTree ` as the parameter

Reply:
I was about to remove that. done